src/app/components/Results/ResultsList.jsx

```diff
@@ -147,7 +147,6 @@ class ResultsList extends React.Component {
               items={items}
               bibId={bibId}
               getRecord={this.getItemRecord}
-              id="search-result-item-table"
```

Comment:
Yes, but I didn't get any issues regarding empty `id`s from aXe or Wave. Did you?